android/app/src/androidTest/java/com/github/quarck/calnotify/dismissedeventsstorage/EventDismissTest.kt

```diff
@@ -487,7 +446,7 @@ class EventDismissTest {
         
         // Also ensure DismissedEventsStorage operations fail
         mockkConstructor(DismissedEventsStorage::class)
-        every { anyConstructed<DismissedEventsStorage>().addEvent(any(), any()) } throws RuntimeException("Storage error")
+        every { anyConstructed<DismissedEventsStorage>().addEvent(any(), any(), any()) } throws RuntimeException("Storage error")
```

Comment:
so we've tried this anyContructed thing and it's a fiasco.

I feel like we might have to fallback to observing the effects of the function while not being able to test as closely.

At least until we refactor out the companion objects

Reply:
fixed it!